ros/lunar/debian/stretch/desktop-full/Dockerfile

```diff
@@ -2,8 +2,11 @@
 # generated from docker_images/create_ros_image.Dockerfile.em
 FROM osrf/ros:lunar-desktop-stretch
 
+# label ros packages
```

Comment:
Not sure if we should change EOL Dockerfile. I not against it, as I like consistency, but we should check for side effects.